enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/DocumentResultEntity.java

```diff
@@ -113,8 +115,7 @@ public class DocumentResultEntity implements Serializable {
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
-        if (!(o instanceof DocumentResultEntity)) return false;
-        DocumentResultEntity that = (DocumentResultEntity) o;
+        if (!(o instanceof DocumentResultEntity that)) return false;
```

Comment:
We can add `final` keyword here, I think.

Reply:
Ok, why not.